internals/build.rs

```diff
@@ -21,10 +24,37 @@ fn main() {
     let minor = version_components
         .next()
         .unwrap_or("0")
-        .parse::<u64>()
+        .parse::<usize>()
```

Comment:
The other option is to change this to `u64`  since assert_eq is used later to compare the two.
```
.map(|minor| minor.parse::<usize>().unwrap_or(0))
```
Not that it matters, but I didn't see any particular reason to prefer u64 over usize here.  I guess there's something to be said for less diff noise.

Reply:
Less diff noise was my point. You could've just used `u64` in `HashSet` below.